docs/data/charts/overview/overview.md

```diff
@@ -1,6 +1,7 @@
 ---
-product: charts
-title: Charts - Overview
+title: React Chart components
```

Comment:
We will see how high we can rank, I have added us in https://app.ahrefs.com/rank-tracker/overview/2944028.